File: CSharp/k03_2.cs
using System;

namespace CSharp
{
    public class K03_2 : IExecutable
    {
        public void Execute()
        {
            try
            {
                Console.Write("身長を入力してください。＞");
                double height = int.Parse(Console.ReadLine());
                Console.Write("体重を入力してください。＞");
                double weight = int.Parse(Console.ReadLine());
                height /= 100.0;
                double standard = height * height * 22.0;
                Console.WriteLine("あなたの標準体重は" + standard + "です。");

                if (weight > standard && (weight - standard) / standard * 100 > 14)
                {
                    Console.WriteLine("太り気味です。");
                }
                else if (weight < standard && (weight - standard) / standard * 100 < -14)
                {
                    Console.WriteLine("痩せ気味です。");
                }
                else
                {
                    Console.WriteLine("普通ですね。");
                }
            }
            catch (Exception ex)
            {
                Console.WriteLine(ex.Message);
                return;
            }

            return;
        }
    }
}

File: Python/k03_2.py
class k03_2(object):
    
    def execute(self):
        try:
            print('身長を入力してください。＞')
            height = float(input())
            height /= 100.0
            print('体重を入力してください。＞')
            weight = float(input())
            standard = height * height * 22.0
            print('あなたの標準体重は{}です。'.format(standard))

            if weight > standard and (weight - standard) / standard * 100 > 14:
                print('太り気味です。')
            elif weight < standard and (weight - standard) / standard * 100 < -14:
                print('痩せ気味です。')
            else:
                print('普通ですね。')

        except ValueError as ve:
            print(ve)